jupyterhub/apihandlers/users.py

```diff
@@ -220,11 +220,11 @@ def post(self, name):
         user = self.find_user(name)
         if user is None:
             raise HTTPError(404, "No such user %r" % name)
-        if user.running:
-            # include notify, so that a server that died is noticed immediately
-            state = yield user.spawner.poll_and_notify()
-            if state is None:
-                raise web.HTTPError(400, "%s's server is already running" % name)
+        #if user.running:
```

Comment:
yes, all of these are going to need to move to the spawner, since singletons no longer make sense. We'll probably still want to keep the properties on the user, mapped to the always-present `self.spawners['']` 'default' Spawner.